### PR TITLE
MAGENTO-2330: simplified tax export for shipping costs

### PR DESCRIPTION
**Pull request by andre-kraus**
# Pull Request Template

## Description

Removing additional calculation for shipping tax amount in order to get rid of issues with edge cases in configurations.

## Type of change

- [x] Bug fix
- [ ] New feature
- [ ] This change requires a documentation update

## Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I updated the CHANGELOG.md

### CODE REVIEW

**Comment by augsteyer** [comment]
you will need to check if there is a test for this and create one if there isn't, all bug fixes/adjustments should have tests. A lot of the times prior it was not necessary is our tests are comprehensive, but I am not sure if there is a test that checks shipping.

File: src/app/code/community/Shopgate/Cloudapi/Helper/Api2/Quote.php [hunk 1]
```diff
@@ -142,13 +142,12 @@ function (Mage_Sales_Model_Quote_Address_Total $total) {
...
 
-        if (isset($totals[self::KEY_SHIPPING]) && $totals[self::KEY_SHIPPING]['value']) {
-            $totals[self::KEY_SHIPPING]['value'] = Mage::helper('tax')->getShippingPrice(
-                $totals[self::KEY_SHIPPING]['value'],
-                Mage::helper('tax')->displayShippingPriceIncludingTax() || Mage::helper('tax')->displayShippingBothPrices(),
-                $quote->getShippingAddress(),
-                $quote->getCustomerTaxClassId()
-            );
+        if (isset($totals[self::KEY_SHIPPING], $totals[self::KEY_SHIPPING]['value'])
+            && $totals[self::KEY_SHIPPING]['value']
```
Comment (augsteyer) [inline src/app/code/community/Shopgate/Cloudapi/Helper/Api2/Quote.php]: may not need to check this as you are not really using this `value` as base for your calculations. I am assuming when $totals[shipping] is set that means some sort of shipping was applied.
A bit of a tricky area as any plugin can rewrite this via their own collector.

File: CHANGELOG.md [hunk 2]
```diff
@@ -5,6 +5,9 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](http://keepachangelog.com/) and this project adheres to [Semantic Versioning](http://semver.org/).
 
 ## [Unreleased]
+### Changed
+- Simplified tax export for shipping costs
```
Comment (augsteyer) [inline CHANGELOG.md]: Maybe better verbiage to use the Changed as a starter:
```suggestion
- Tax export for shipping costs to make them simpler
```

File: src/app/code/community/Shopgate/Cloudapi/Helper/Api2/Quote.php (hunk 1 above)
Comment (andre-kraus) [reply src/app/code/community/Shopgate/Cloudapi/Helper/Api2/Quote.php]: true that.

**Pull request merged**